### ISSUE TITLE
Need to disallow direct content object delete

### ISSUE THREAD

**Issue by paulgoleary** [eluv-io/contracts#29]
Right now the owner of a content object can directly call `kill` on the contract. This corrupts the `elv-master` data model because we need the `ContentObjectDeleted` event to be thrown from the owning library in order to keep indexes in sync.

However, we can't immediately just disallow any account besides the library to call `kill` because - for other reasons - we took `onlyOwner` off of the delete content function in the library.

Long story short, we need to figure out the correct way to auth this.

**Comment by elv-preethi** [comment]
tested `PM/qdel` branch contract with `qfab_cli library content delete` command with following scenarios :
* content and library created by same user -> success
* content created by different user. -> if the some one other than content owner tries to access it, throws error.
